landoscript/src/landoscript/actions/merge_day.py

```diff
@@ -183,7 +183,12 @@ async def run(
     with open(os.path.join(public_artifact_dir, "replacements.diff"), "w+") as f:
         f.write(diff)
 
-    commitmsg = "No Bug - Update configs after merge day operations"
+    commitmsg = "No Bug - Update configs after merge day operations a=release"
+    # keep the uglier magic strings out of the first line
+    # also put them on their own lines to make it clear that they are
+    # distinct from one another
+    commitmsg += "\nIGNORE BROKEN CHANGESETS"
```

Comment:
```suggestion
    commitmsg += "\n\nIGNORE BROKEN CHANGESETS"
```

Add an empty line between the summary line and the rest of the commit message